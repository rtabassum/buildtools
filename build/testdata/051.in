foo(name='foo',
  srcs=[bar])

aaa
bbb
ccc

def function(
    # This shouldn't mess with indendation counting
    x, y = z
):
  aaa
  bbb
  def g(a, s, d):
    ccc
# Misalingned comments
      # are not a syntax error
    ddd
    def h(a, s = d):   # this is h
      eee

    def i():
      def j():
        return h([1],
                 [2]) # returns h

  return u()

bar
def a():
  return
