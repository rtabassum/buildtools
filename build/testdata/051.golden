foo(
    name = "foo",
    srcs = [bar],
)

aaa

bbb

ccc

def function(
    # This shouldn't mess with indentation counting
    x, y = z
):
  aaa
  bbb
  def g(a, s, d):
    ccc
# Misalingned comments
      # are not a syntax error
    ddd
    def h(a, s = d):   # this is h
      eee

    def k(g): g += 1; return g

    def i():
      def j():
        return h([1],
                 [2]) # returns h

  return u()

bar

def a():
  return
